bin/webpack.js

```diff
@@ -382,6 +388,10 @@
 				}
 			});
 
+			ifArg("info-verbosity", function(value) {
+				outputOptions.infoVerbosity = value;
```

Comment:
If `info-verbosity` is not a string, i.e boolean, it will still output. Would be nice with an error that will match against `webpack-schema`. https://github.com/webpack/webpack/blob/master/schemas/webpackOptionsSchema.json

You should send a PR to add `info-verbosity` and link it to this.

Reply:
I will modify within this scope (the schema), compeltely forgot about it

Reply:
schema added and pr's linked in description section

Reply:
Cool beans 🐻